### PR TITLE
project in progress...

### PR DESCRIPTION
**Pull request by soniaroh**
used an npm library to generate the outline of the carousel, (will have to double check with Gwyn to see if thats allowed) was able to get data to show in the carousel from the the database..

### CODE REVIEW

**Comment by avincenthill** [comment]
Recommend following this guide for phrasing commit messages in imperative tense: https://chris.beams.io/posts/git-commit/ ... ie this code change will "commit message"

File: client/src/NextNav.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function NextArrow(props) {
+	const { className, style, onClick } = props;
+	return <div className={className} style={{ ...style, background: 'black' }} onClick={onClick} />;
```
Comment (avincenthill) [inline client/src/NextNav.jsx:5]: Cool

File: client/src/SlideShow.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	componentDidMount() {
+		this.getProducts();
+	}
+
+	// using fetch to get data from database.. might refactor to AJAX later
+
+	getProducts() {
+		fetch('http://localhost:4043/products')
```
Comment (avincenthill) [inline client/src/SlideShow.jsx:22]: use env vars handled by dotenv, ie process.env.url here

**Review by michchen58 — APPROVED** [review]
looks great

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+let mysql = require('mysql');
+var faker = require('faker');
+
+
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'password',
+    database: 'slideShowData'
```
Comment (avincenthill) [inline database/index.js:9]: protect sensitive data like this by gitignoring .env and handling with npm dotenv package

File: server/dotenv.env [hunk 4]
```diff
@@ -0,0 +1,3 @@
+DB_HOST=localhost
+DB_USER=root
+DB_PASS='password'
```
Comment (avincenthill) [inline server/dotenv.env:3]: gitignore this file and rename to .env, put in root dir

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        } else {
+            return res.json({
+                data: results
+            })
+        }
+    });
+});
+
+
+const PORT = 4043;
```
Comment (avincenthill) [inline server/index.js:38]: use env var here

**Review by avincenthill — APPROVED** [review]
Needs change to handling of env vars, make sure to not commit usernames/pws, else looks good!

File: client/src/SlideShow.jsx (hunk 2 above)
Comment (soniaroh) [reply client/src/SlideShow.jsx:22]: added process.env.url

File: database/index.js (hunk 3 above)
Comment (soniaroh) [reply database/index.js:9]: password was removed.

File: server/index.js (hunk 5 above)
Comment (soniaroh) [reply server/index.js:38]: added process.env.url

**Pull request merged**